apps/crm/src/modules/providers/providers-list/providers-list.tsx

```diff
@@ -0,0 +1,60 @@
+import { ProvidersTypes, ProviderType } from "../providers-types";
+import MailchimpLogo from "../../../assets/mailchimp.svg";
+import SegmentLogo from "../../../assets/semgent.png";
+import RudderstackLogo from "../../../assets/rudderstack.png";
+import { HTMLAttributes } from "react";
+import clsx from "clsx";
+import styles from "./providers-list.module.css";
+import { Text, Box, MarketplaceIcon } from "@saleor/macaw-ui/next";
+import { TextLink } from "../../ui/text-link/text-link";
+
+type Props = {
+  onProviderClick(provider: ProviderType): void;
+  activeProvider?: ProviderType;
+} & HTMLAttributes<HTMLUListElement>;
+
+export const ProvidersList = ({ className, onProviderClick, activeProvider, ...props }: Props) => {
+  return (
+    <ul className={clsx(className, styles.list)}>
```

Comment:
**question**: why you decided to use CSS modules here? 

Is it something that the current MacawUI lacks?

Reply:
Technically this component was older, but when I look at it - probably not.
However:
- I feel somehow worse DX when I have to add *a lot* of rules - probably this will be gone if I have a good pre-defined component like this.
- Some of rules are strange to use, e.g. I can't easily add border - I have to add 3 rules (color, size, style) - can we have 1 rule?
- There are some missing rules, like I can't set `height: 50`, I need to do `__height` which looks a little hacky, considering it adds inline style

But I will revisit this component once more